usr/share/rear/prep/NETFS/default/05_check_NETFS_requirements.sh

```diff
@@ -66,6 +65,10 @@ case "$(basename $BACKUP_PROG)" in
 esac
 
 # include required programs
+# FIXME: the code below includes mount.* and umount.* programs for all non-empty schemes
+# i.e. for any non-empty BACKUP_URL like usb tape file sshfs ftpfs
+# and it includes 'mount.' for empty schemes (e.g. if BACKUP_URL is not set)
+# but I <jsmeix@suse.de> cannot decide if there is a subtle reason for that
```

Comment:
@jsmeix a catch all rule so we do not miss any important executable needed a certain scheme. It does not hurt I would say.